other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,126 @@
+# Launching a Site on Altis
+
+The process for launching a site on Altis varies depending on the type of project. Nonetheless, there are common steps that need to be done no matter the context and this guide will walk you through what needs to be done in preparation for your site launch.
+
+## Planning
+
+Create a launch issue in the project repository. This issue should include:
+
+* Client name
+* Client business region
+* Client contact
```

Comment:
This could be followed by an internal team for a given client in which case this isn't needed. Additionally I think now that this issue isn't something the product team would need to interact with directly this information should be implicit from both the project team and Altis side so we can drop it.